cmdutil/debug/debug.go

```diff
@@ -85,37 +85,19 @@ type PProfServer struct {
 	pprofServer *http.Server
 }
 
-// ProfileConfig holds the configuration for the pprof server.
-type PProfServerConfig struct {
-	Addr                 string
-	MutexProfileFraction int
-}
-
-// defaultMutexProfileFraction is the default value for MutexProfileFraction
-const defaultMutexProfileFraction = 2
-
 // NewPProfServer sets up a pprof server with configurable profiling types and returns a PProfServer instance.
-func NewPProfServer(config PProfServerConfig, l logrus.FieldLogger) *PProfServer {
-	if config.Addr == "" {
-		config.Addr = "127.0.0.1:9998" // Default port
-	}
-
-	// Use a local variable for the mutex profile fraction
-	mpf := defaultMutexProfileFraction
-	if config.MutexProfileFraction != 0 {
-		mpf = config.MutexProfileFraction
-	}
-	runtime.SetMutexProfileFraction(mpf)
+func NewPProfServer(l logrus.FieldLogger, port int, mutexProfileFraction int) *PProfServer {
```

Comment:
Let's standardize on whether or not the second "p" is capitalized. Either `Pprof` or `PProf`